src/security/engines/BrowserKeyStore.js

```diff
@@ -122,19 +128,23 @@ class BrowserKeyStore {
             getReq.onsuccess = () => {
                 const member = getReq.result;
                 if (!member) {
-                    reject(new Error(`member ${memberId} not found`));
+                    return reject(new Error(`member ${memberId} not found`));
                 }
-                Object.values(member).forEach((level) => {
-                    if (level.id === keyId) {
+                Object.values(member).forEach((keypair) => {
+                    if (keypair.id === keyId) {
+                        if (keypair.expiresAtMs < Date.now()) {
+                            reject(new Error(`Key with id ${keypair.id} has expired`));
```

Comment:
I only added return in places where not returning would potentially cause getting property of undefined. Adding return changes the behavior slightly, e.g. here it would make the active member not get set if all of the members keys are expired. Is that behavior desirable?

Reply:
hmm i see, this is probably fine.
also can you camel case keyPair just to be consistent with others